src/test/regress/sql/multi_fdw_shardstorage.sql

```diff
@@ -0,0 +1,40 @@
+--
+-- MULTI_FDW_SHARDSTORAGE
+--
+
+-- Create two tables one regular and one foreign, then check whether
+-- shardstorage is correct
+
```

Comment:
It would be nice to set shardIDs explicitly before the tests referencing specific shard IDs with something like:
`ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 400000;`